app/client/src/utils/Analytics/smartlook.ts

```
@@ -0,0 +1,19 @@
+
+  public static init() {
+    const {
+      smartLook: { enabled, id },
+    } = getAppsmithConfigs();
+
+    if (enabled && !this.initialised) {
+      smartlookClient.init(id);
+      this.initialised = true;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)